fennel/internal_lib/schema/test_schema.py

```diff
@@ -799,3 +803,158 @@ def test_convert_dtype_to_arrow_type():
         str(arrow_dtype)
         == "list<item: struct<a: int64, b: map<string, int64>, c: list<item: int64>>>"
     )
+
+
+class TestDataTypeConversions(unittest.TestCase):
+
+    def test_int_type(self):
+        original = int
+        proto = get_datatype(original)
+        result = from_proto(proto)
+        self.assertEqual(original, result)
```

Comment:
Super nit but this would be better written as case builder pattern. Where we have a list of lots of cases and for each case, we make a simple check in a for loop. Makes it easy to maintain/add more tests with low overhead.